core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -104,23 +111,15 @@ <h2 ng-if="$ctrl.collectionPlaythrough.hasFinishedCollection()" class="oppia-pag
                     stroke="#006553"
                     stroke-width="2"/>
           </a>
-          <text x="50%"
-                y="15"
-                alignment-baseline="middle"
-                font-family="Capriola, Roboto, Arial, sans-serif"
-                font-size="20"
-                fill="#006553"
-                text-anchor="middle">
-            <[$index + 1]>
-          </text>
         </svg>
       </a>
     </div>
 
     <div ng-if="$ctrl.collection" class="oppia-collection-table hidden-md hidden-lg hidden-xl">
       <img ng-src="<[$ctrl.getStaticImageUrl('/general/collection_mascot.svg')]>" class="mobile-lesson-icon">
       <div class="mobile-path-segment" ng-repeat="node in $ctrl.collection.getCollectionNodes()" id="mobile-path-anchor-<[$index]>">
-        <a href="" style="position: absolute; left: 50%; transform: translate(-50%, 195px); z-index: 1;">
+        <a href="" style="position: absolute; left: 50%; transform: translate(-50%, 195px); z-index: 1;text-align: center">
+          <div ng-style="{color:'#006553','font-size':'20px'}">{{node._explorationSummaryObject.title}}</div>
```

Comment:
Convert this to `style` since there isn't anything here that is dynamic.

Reply:
Yup Sure

Reply:
Done!